src/util.ts

```diff
@@ -0,0 +1,18 @@
+export function isBase64(str: string): boolean {
+  if (!str) return false; // Handle empty string case
+
+  try {
+    const base64Regex =
```

Comment:
Technically, this regex is overkill for checking a certain charset + length. Also not sure if Zerion is using this flavor of base64 (`/` and `+`) or URL flavor (`-` and `_`)

Reply:
Good point... there... do you have a suggestion for an alternate pattern?